transports/transport-rest/transport-rest-vertx/src/main/java/io/servicecomb/transport/rest/vertx/accesslog/AccessLogConfiguration.java

```diff
@@ -0,0 +1,38 @@
+package io.servicecomb.transport.rest.vertx.accesslog;
+
+import com.netflix.config.DynamicPropertyFactory;
+
+import io.servicecomb.core.BootListener.BootEvent;
+
+public final class AccessLogConfiguration {
+  private static final String BASE = "cse.accesslog.";
```

Comment:
This should be configurable.

Reply:
This is the name prefix of configuration item. It has been renamed to "servicecomb.accesslog."